packages/FormElement/stories/variations/Variations.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import Heading from "@paprika/heading";
+import { Rule } from "storybook/assets/styles/common.styles";
+import { FormElementStory } from "../FormElement.stories.styles";
+import CheckboxExample from "../examples/CheckboxExample";
+import RadioExample from "../examples/RadioExample";
+import DatePickerExample from "../examples/DatePickerExample";
+import AccessibilityExample from "../examples/AccessibilityExample";
+import ListBoxExample from "../examples/ListBoxExample";
+import ButtonGroupExample from "../examples/ButtonGroupExample";
+import NativeInputExample from "../examples/NativeInputExample";
+import NestedExample from "../examples/NestedExample";
+import HTMLExample from "../examples/HTMLExample";
+import ComponentLabelExample from "../examples/ComponentLabelExample";
+
+export default function Variations() {
+  return (
+    <FormElementStory>
+      <Heading level={1} displayLevel={2} isLight>
+        Variations
+      </Heading>
+      <Rule />
+      <AccessibilityExample />
+      <Rule />
+      <CheckboxExample />
+      <Rule />
+      <RadioExample />
+      <Rule />
+      <DatePickerExample />
+      <Rule />
+      <ListBoxExample />
+      <Rule />
+      <ButtonGroupExample />
+      <Rule />
+      <NativeInputExample />
+      <Rule />
+      <NestedExample />
+      <Rule />
+      <HTMLExample />
+      <Rule />
+      <ComponentLabelExample />
```

Comment:
I see where you're going with this... trying to re-use the code as much as possible.  I admire the intent, but the Variations story should be more about showing the different permutations of the actual `<FormLabel>`, less about the different use cases.  So, I think you could show a bunch of examples with error states or not... helper/description text or not... disabled, inline, required, etc. Basically all the combinations you can get with the Showcase, but at a glance.  But for this story, I don't think the actual input field matters.  It can just be an `<Input>` for all the variations. Make sense?